resources/js/common/components/+vendor/BaseDrawer.tsx

```diff
@@ -74,7 +74,7 @@ const BaseDrawerHeader = ({ className, ...props }: React.HTMLAttributes<HTMLDivE
 BaseDrawerHeader.displayName = 'BaseDrawerHeader';
 
 const BaseDrawerFooter = ({ className, ...props }: React.HTMLAttributes<HTMLDivElement>) => (
-  <div className={cn('mt-auto flex flex-col gap-2 p-4', className)} {...props} />
+  <div className={cn('mt-auto flex flex-col gap-2 px-4 pb-6 pt-4', className)} {...props} />
```

Comment:
Should the drawer height be adjusted to account for the change ? It's slightly obscuring the top bar now:
![image](https://github.com/user-attachments/assets/bff6cb7c-8788-422c-acf2-23523b89a214)

before:
![image](https://github.com/user-attachments/assets/65785f15-71b2-4159-9842-d7b9d5576822)

Or is that just coincidental?

Reply:
Good question to ask. Coincidental and likely should not be changed - this will unfortunately be something that varies on a device-by-device basis.